Simplified/Book/UI/NYPLBookCellDelegate.m

```diff
@@ -405,41 +425,70 @@ - (void)didSelectCancelForBookDownloadingCell:(NYPLBookDownloadingCell *const)ce
 
 #pragma mark Audiobook Manager Refresh Delegate
 
-- (void)audiobookManagerDidRequestRefresh {
+- (void)audiobookManagerDidRequestRefresh
+{
+  // while `audiobookManagerDidRequestRefresh` is guaranteed to be called
+  // always on the main thread, this lock prevents us from starting the book
+  // download a second time before reaching a completion for the first.
   if (![self.refreshAudiobookLock tryLock]) {
     return;
   }
     
   [[NYPLBookRegistry sharedRegistry] setState:NYPLBookStateDownloadNeeded forIdentifier:self.book.identifier];
 
 #if FEATURE_OVERDRIVE
-  [[NSNotificationCenter defaultCenter] addObserver:self selector:@selector(updateODAudiobookManifest) name:NSNotification.NYPLMyBooksDownloadCenterDidChange object:nil];
+  // TODO: Configure proxy so that audiobook download fails
```

Comment:
how do you trigger this method (`audiobookManagerDidRequestRefresh`)? I have been unable to see it called...

Reply:
[here](https://github.com/NYPL-Simplified/NYPLAudiobookToolkit/blob/master/NYPLAudiobookToolkit/UI/AudiobookPlayerViewController.swift#L681) in NYPLAudiobookToolkit
We are doing some checks in there and it should be Overdrive exclusive. Not sure how a `FindawayAudiobook` got through that.